sourcegraph/cody
# Onboarding: Add interactive walkthrough

## Description

part of #3401

This PR adds an interactive walkthrough (see demo)


https://github.com/sourcegraph/cody/assets/9516420/244d14f1-329b-4f47-b5f0-225d68c7beb6


## Test plan

1. Click "Tutorial" in the Cody sidebar
2. Check the tutorial works as expected
3. Check autocomplete/chat/edit functions as expected outside of the tutorial

Note: This is feature flagged off right now, so there is low risk to this change
